### DISCUSSION TITLE
移动端的导航栏显示文字没有对齐

### DISCUSSION THREAD

**Discussion by yeung66** [liuyib/hexo-theme-stun#146] [Q&A（反馈问题、提问、报告 Bug 等）] [answered]
似乎是因为 icons 的宽度不一致导致文字也没有对齐？

![image](https://user-images.githubusercontent.com/22522618/103145808-1394cb00-477b-11eb-8840-ccce5d806eee.png)

**Comment by liuyib** [comment, marked as answer]
这个问题后面会优化，感谢你的反馈 :heart:

**Reply by hyggevv** [reply]
想问问您；阅读全文的功能是需要配置嘛

**Reply by liuyib** [reply]
> 想问问您；阅读全文的功能是需要配置嘛

不需要配置，需要改造你的 md 文件

**Comment by hyggevv** [comment]
想问问你们有没有阅读全文的功能；我用这个主题没有阅读全文的功能

**Reply by liuyib** [reply]
> 想问问你们有没有阅读全文的功能；我用这个主题没有阅读全文的功能

https://theme-stun.github.io/docs/zh-CN/guide/primary.html#%E6%96%87%E7%AB%A0%E6%91%98%E8%A6%81

![image](https://user-images.githubusercontent.com/38221479/234753285-853766d3-9d30-43c0-a43e-626b5557b040.png)


有了文章摘要，就会显示出“阅读全文”按钮
